tasks/eth/013-fp-upgrade-fjord/.env

```diff
@@ -0,0 +1,9 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+SUPERCHAIN_CONFIG_ADDR=0x95703e0982140d16f8eba6d158fccede42f04a4c
+OWNER_SAFE=0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+
+SAFE_NONCE_0X5A0AAE59D09FCCBDDB6C6CCEB07B7279367C3D2A=4
+SAFE_NONCE_0XC2819DC788505AAC350142A7A707BF9D03E3BD03=5
+SAFE_NONCE_0X847B5C174615B1B7FDF770882256E2D3E95B9D92=4
```

Comment:
Can you add a comment to justify each nonce here so it's easier to verify (i.e. based on the current nonce + expected transactions to execute before)?

Reply:
Fixed. Though please confirm that the L2 POA change does not require the L1 Foundation Safe. The nonces here assume the safe nonces will remain unchanged at signing.

Reply:
My checks showed that the last one (0x847B5c174615B1B7fDF770882256e2D3E95b9D92) should be 5. I went through the [sequencing table](https://www.notion.so/oplabs/4e021daa38444d8ebda3770a43c48a6b?v=7b9ecb4d958044c6b16ccbd9a195519e&pvs=4) and filled in the Hardcoded Nonce table, can you confirm? 

> Though please confirm that the L2 POA change does not require the L1 Foundation Safe. 

What did you mean here? The L2 PAO change was already completed as part of upgrade 8

Reply:
You're right. I was looking at a totally different signing ceremony table.

Reply:
Fixed. Thanks for confirming this!